.github/workflows/ci.yml

```diff
@@ -7,7 +7,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        java: [ 'adopt@1.8' ] # TODO: add test for adopt@1.11 and possibly adopt@1.16
+        java: [ 'adopt@1.8', 'adopt@1.11', 'openjdk@1.11' ]
```

Comment:
How about adding 16 to the mix?

Reply:
I couldn't find it in the list of java versions when unfolding the scala-setup action. But I can try to add it

Reply:
It doesn't seem to be available currently: ` Error: Couldn't find Java openjdk@1.16.` Or do you think I missed something?

Reply:
Yeah, it seems like there's only Zulu, so you could try that one: `zulu@1.16`